### PR TITLE
[HW 9.1]: Validation of Micropost and association

### PR DESCRIPTION
**Pull request by aonufrei**
* Added validation for `Micropost` model ( minimum and maximum length of micropost content and required user_id to present ).
* Added relationship between `Micropost` and `User` models ( one user has multiple microposts and micropost belongs to user ).

### CODE REVIEW

File: app/models/micropost.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 class Micropost < ApplicationRecord
+    belongs_to :user
+
+    validates :content, length: {
+        minimum: 3,
+        maximum: 300
```
Comment (NickUkraine24) [inline app/models/micropost.rb]: If I'm not mistaken, `maximum` was `140`.

**Review by NickUkraine24 — APPROVED** [review]
Please, resolve the problem about committing `.idea` files.

File: app/models/micropost.rb (hunk 1 above)
Comment (aonufrei) [reply app/models/micropost.rb]: > Please, resolve the problem about committing `.idea` files.

I don't have .idea files in my project, maybe you're mistaken? I use vscode

Comment (aonufrei) [reply app/models/micropost.rb]: > If I'm not mistaken, `maximum` was `140`.

Ok, I'll change it

**Comment by NickUkraine24** [comment]
That's a good start to improve your skills in Ruby on Rails.=)

**Pull request merged**